### PR TITLE
Add DSTU 2 documentation for DiagnosticReport.

### PR DESCRIPTION
**Pull request by davischristi**

### CODE REVIEW

**Comment by cerbs** [comment]
:+1:

**Comment by MaxPhilips** [comment]
+1

**Comment by davischristi** [comment]
Rebased on master, Merging

File: content/dstu2/diagnostic-report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 BALLOT API
+---
+
+# Immunization
```
Comment (MaxPhilips) [inline content/dstu2/diagnostic-report.md]: @davischristi We all missed this when reviewing the PR, but the title here should be Diagnostic Report

Comment (davischristi) [inline content/dstu2/diagnostic-report.md]: Thanks for pointing that out @MaxPhilips Issued a PR to fix this: https://github.com/cerner/fhir.cerner.com/pull/48

**Pull request merged**